documentation/manage/updating-your-extension.md

```diff
@@ -0,0 +1,216 @@
+| `update_info_url`         | `string` (Optional)               | A link to an HTML file containing information about the update.                                                                                                                                                                                                                                                                                                                                                                                           |
+| `multiprocess_compatible` | `bool` (Optional) (default: true) | If false, this add-on requires compatibility shims to run in a [multi-process Firefox](https://developer.mozilla.org/docs/Mozilla/Firefox/Multiprocess_Firefox) environment.                                                                                                                                                                                                                                                                              |
+| `applications`            | `object` (Optional)               | An object containing application-specific compatibility information. Each property must contain an application object, as described below. The only application currently supported is `gecko`, which includes Firefox, and all other applications built on the same runtime. If this property is omitted, support for Gecko is assumed. Otherwise, if this property is defined, it must contain a `gecko` property, or the update entry will be ignored. |
+
+{% endcapture %}
+{% include modules/table.html
+	content=table
+%}
+
+<!-- END: Table -->
+
+Application objects specify compatibility information for a specific application. They must be object literals, and may have the following properties:
+
+<!-- Table -->
+
+{% capture table %}
+
+| Property               | Type                                  | Description                                                                                                     |
+| ---------------------- | ------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| `strict_min_version`   | `string` (Optional) (default: 42.0a1) | The minimum version of the application this add-on will run on.                                                 |
+| `strict_max_version`   | `string` (Optional) (default: \*)     | The maximum version of the application this add-on will run on.                                                 |
+| `advisory_max_version` | `string` (Optional) (default: \*)     | The maximum version of the application this add-on is likely to run on. This property is ignored in most cases. |
+
+{% endcapture %}
+{% include modules/table.html
+	content=table
+%}
+
+<!-- END: Table -->
+
+## Testing Automatic Updating
```

Comment:
Start new section here and use h2 (##) for "Testing Automatic Updating". Section ID should be testing-automatic-updating